.github/templates/linux_binary_build_workflow.yml.j2

```diff
@@ -48,25 +52,43 @@ jobs:
 {%- for config in build_configs %}
   !{{ config["build_name"] }}-build:
     if: ${{ github.repository_owner == 'pytorch' }}
+    {%- if "aarch64" in build_environment %}
+    uses: ./.github/workflows/_binary-build-aarch64.yml
+    {%- else %}
     uses: ./.github/workflows/_binary-build-linux.yml
+    {%- endif %}
     with:!{{ upload.binary_env_as_input(config) }}
+      {%- if "aarch64" in build_environment %}
+      runs_on: linux.t4g.2xlarge
```

Comment:
Please keep only first statemetn here for aarch64

Reply:
updated as requested. :)